core-domain/src/main/java/org/uniprot/core/unirule/builder/CaseRuleBuilder.java

```diff
@@ -0,0 +1,46 @@
+package org.uniprot.core.unirule.builder;
+
+import static org.uniprot.core.util.Utils.nullThrowIllegalArgument;
+
+import java.util.List;
+
+import javax.annotation.Nonnull;
+
+import org.uniprot.core.unirule.Annotation;
+import org.uniprot.core.unirule.CaseRule;
+import org.uniprot.core.unirule.ConditionSet;
+import org.uniprot.core.unirule.RuleException;
+import org.uniprot.core.unirule.impl.CaseRuleImpl;
+
+public class CaseRuleBuilder<R> extends AbstractRuleBuilder<CaseRule<R>, R> {
+    private boolean overallStatsExempted;
+
+    public CaseRuleBuilder(
+            List<ConditionSet> conditionSets,
+            List<Annotation> annotations,
+            List<RuleException<R>> ruleExceptions) {
+        super(conditionSets, annotations, ruleExceptions);
```

Comment:
correct me if i am wrong
business say
you can't create a rule until you supply these three lists inside constructor

Reply:
No. I fixed it. Thanks for finding it.